Documentation/advanced-configuration.md

```diff
@@ -413,3 +413,72 @@ other OSDs holding replica data are unavailable:
 ```bash
 ceph osd primary-affinity osd.0 0
 ```
+
+### OSDs not starting when the Ceph cluster is not healthy
```

Comment:
Hi @lorenz, thanks for this.  Could you add your documentation under a new section (`##` heading) and add it to the table of contents (`[OSDs not starting when the Ceph cluster is not healthy
](#osds-not-starting-when-the-ceph-cluster-is-not-healthy)`)?

If we document more of these issues/fixes they could go under a more general section in the future.